doc/manual/source/man/ldns-signzone.rst

```diff
@@ -22,75 +23,143 @@ Arguments
 
       The zonefile to sign.
 
+      Note: Unlike the original LDNS, any existing NSEC(3), NSEC3PARAM and/or
+      RRSIG resource records will be skipped when loading the zonefile.
+
 .. option:: <KEY>...
 
       The keys to sign the zonefile with.
 
+      Keys must be specified by their base file name (usually
+      ``K<name>+<alg>+<id>``), i.e. WITHOUT the ``.private`` or ``.key``
+      extension, with an optional path prefix. The ``.private`` file is
+      required to exist. The ``.key`` file will be used if a ``DNSKEY`` record
+      corresponding to the ``.private`` key cannot be found.
+
+      Multiple keys can be specified. Key Signing Keys are used as such when
+      they are either already present in the zone, or specified in a ``.key``
+      file, and have the Secure Entry Point flag set.
+
+      Note: Unlike the original LDNS, DNSKEY algorithms marked as ``MUST NOT``
+      or ``NOT RECOMMENDED`` in table 3.1 of RFC 8624 "Algorithm
+      Implementation Requirements and Usage Guidance for DNSSEC" are NOT
+      supported.
```

Comment:
```suggestion
      Note: Unlike the original LDNS:
        - Public keys corresponding to ``.private`` key MUST be supplied,
          either as DNSKEY RRs in the given zone or as ``.key`` files. This
          Implementation is not able to generate missing public keys.
        - DNSKEY algorithms marked as ``MUST NOT`` or ``NOT RECOMMENDED`` in
          table 3.1 of RFC 8624 "Algorithm Implementation Requirements and
          Usage Guidance for DNSSEC" are NOT supported.
```

Reply:
I think that text is now wrong. I added support for RSASHA512 to domain for some reason.